acbuild/copy.go

```diff
@@ -20,9 +20,9 @@ import (
 
 var (
 	cmdCopy = &cobra.Command{
-		Use:     "copy PATH_ON_HOST PATH_IN_ACI",
+		Use:     "copy PATH_ON_HOST... PATH_IN_ACI",
```

Comment:
Maybe with some serious tweaking of the help page templates.

In their current state we get:

```bash
USAGE:
	acbuild copy PATH_ON_HOST PATH_IN_ACI
copy PATH_ON_HOST... DIRECTORY_IN_ACI
```